packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Deposit/EnterAmount/template.success.tsx

```diff
@@ -122,7 +122,7 @@ const LimitSection = ({ fiatCurrency, paymentMethods }) => {
     (method) => method.type === SBPaymentTypes.BANK_TRANSFER
   )
 
-  if (bankTransfer?.limits) {
+  if (bankTransfer?.limits?.daily) {
```

Comment:
@blockdylanb I double checked with Lorenzo this part was not changed but I do not see anything daily related just min/max inside limits

Reply:
I think we shouldn't have the `?.daily` here on second thought. It's in prod api now, but they want to remove it so then this code will never execute.